scripts/services/sources-api.sh

```diff
@@ -4,4 +4,4 @@ source config.sh
 source init-common.sh
 
 cd ${SOURCES_API_DIR}
-env PORT=${SOURCES_PORT} PATH_PREFIX="api" APP_NAME="sources" rails s --binding 0.0.0.0
+env PORT=${SOURCES_PORT} SOURCES_PSK=${SOURCES_PSK} PATH_PREFIX="api" APP_NAME="sources" rails s --binding 0.0.0.0
```

Comment:
I guess I have to define and export SOURCES_PSK in my config file,  if so, can it be mentioned in readme or somewhere ? thanks!

Reply:
Sure, added to the README.